docs/content/modules/tensor_mechanics/systems/Materials/RecomputeRadialReturn.md

```diff
@@ -1,11 +1,78 @@
-<!-- MOOSE Object Documentation Stub: Remove this when content is added. -->
-#RecomputeRadialReturn
+#Recompute Radial Return
+!description /Materials/RecomputeRadialReturn
+
 !devel /Materials/RecomputeRadialReturn float=right width=auto margin=20px padding=20px background-color=#F8F8F8
 
-!description /Materials/RecomputeRadialReturn
+###Algorithm References
+The radial return mapping method, introduced by Simo and Taylor (1985), uses a von Mises yield surface to determine the increment of plastic strain necessary to return the stress state to the yield surface after a trial stress increment takes the computed stress state across the yield surface.  Because the von Mises yield surface in the deviatoric stress space has the shape of a circle, the _plastic correction stress_ is always directed towards the center of the yield surface circle.
+
+In addition to the Sumo and Hughes (1998) textbook, _Introduction to Computational Plasticity_ by Dunnu and Petrinic (2004) (ISBN: 0-19-856826-6) is an excellent reference for users working with the Recompute Radial Return materials; several of the isotropic plasticity and creep effective plastic strain increment algorithms are taken from Dunne and Petrinic.
+
+###The Recompute Radial Return Description
+The recompute materials are not called by MOOSE directly but instead only by other materials using the `computeProperties` method.  For the `RecomputeRadialReturn` materials, this calling material is [ComputeReturnMappingStress](auto::ComputeReturnMappingStress).  Separating the call to the recompute materials from MOOSE allows us to iteratively call the recompute materials as is required to achieve convergence.
+
+##Radial Return Algorithm Overview
+
+
+In this numerical approach, a trial stress is calculated at the start of each simulation time increment; the trial stress calculation assumed all of the new strain increment is elastic strain:
+$$
+\sigma_{trial} = C_{ijkl} \left( \Delta \epsilon_{assumed-elastic} + \epsilon_{elastic}^{old} \right)
+$$
+
+!image tensor_mechanics-RadialReturnStressSpace.png width=300 float=right caption=Figure 1: A trial stress is shown outside of the devatoric yield surface and the radial return stress which is normal to the yield surface.
```

Comment:
@milljm Here's that really long figure caption I promised as a test case for the css template.  Alternatively I can strive to be less verbose